### DISCUSSION TITLE
FileUpload don't save to DB

### DISCUSSION THREAD

**Discussion by maverik23** [filamentphp/filament#13611] [Help]
### Package

Form builder

### Package Version

3.2

### How can we help you?

i have a simple code to upload user avatar.

`FileUpload::make('avatar', 'profile_photo_path')
                    ->maxSize(1024)
                    ->image()
                    ->avatar()
                    ->imageEditor()
                    ->circleCropper()
                    ->disk('public')
                    ->directory('user_avatar'),`
                    
user model

`protected $fillable = [
        'name',
        'email',
        'password',
        'profile_photo_path',
        'email_verified_at'
    ];`

the file upload to the correct folder, but path is not stored in DB

**Comment by leandrocfe** [comment]
`FileUpload::make('profile_photo_path')...` ?

**Reply by maverik23** [reply]
Of course you are right, I don't know why I want to put the label() in the make().

My bad! sorry
